neuraxle/pipeline.py

```diff
@@ -70,7 +70,7 @@ def transform(self, data_inputs: Any):
         )
         data_container = DataContainer(current_ids=current_ids, data_inputs=data_inputs)
 
-        context = ExecutionContext.from_root(self, self.cache_folder)
+        context = ExecutionContext.from_root(ExecutionMode.TRANSFORM, self, self.cache_folder)
```

Comment:
I'd reorder the order of the arguments to feel more natural. Like: 

`context = ExecutionContext.from_root(self, ExecutionMode.TRANSFORM, self.cache_folder)`
or something else.

Reply:
If possible, because default arguments may force the first one to be first....